src/main/java/de/uhd/ifi/se/decision/management/jira/view/treeviewer/TreeViewer.java

```diff
@@ -88,13 +88,22 @@ public TreeViewer(FilterSettings filterSettings) {
 		KnowledgeElement rootElement = filterSettings.getSelectedElement();
 
 		if (rootElement != null) {
```

Comment:
What happens when the root element is null?

Reply:
I used the return statement at the end of the if-statement. I think the code is easier to read like this but that might not be the case.